src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -171,7 +391,7 @@ sentry_sdk.init(
 
 **Manual Instrumentation**
 
-To manually instrument certain regions of your code, you can create a [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction) to capture them.
+To manually instrument certain regions of your code, you can create a transaction to capture them.
```

Comment:
Should we explain here that `start_span` actually starts a transaction, and that `item.get_transaction()` would be some sort of string representing the `item`?

I'd even update the code snippet to look more like:

```python
import sentry_sdk

while True:
    item = get_from_queue()
    with sentry_sdk.start_span(op="task", transaction="process.{}".format(item.description)):
        # process_item may create more spans internally (see next examples)
        process_item(item)
```

Reply:
None of this section has been touched. That's the next project. 😊

So, we're on the same page (metaphorically as well as literally!), but that's for another PR.